dexbot/strategies/king_of_the_hill.py

```diff
@@ -0,0 +1,376 @@
+# Python imports
+from datetime import datetime, timedelta
+
+# Project imports
+from dexbot.controllers.strategy_controller import StrategyController
+from dexbot.strategies.base import StrategyBase, ConfigElement, DetailElement
+from dexbot.qt_queue.idle_queue import idle_add
+
+# Third party imports
+from bitshares.market import Market
+
+STRATEGY_NAME = 'King Of The Hill'
+
+
+class Strategy(StrategyBase):
+    """ King Of The Hill strategy
```

Comment:
Change capitalization to `King of the Hill` here as well, and other places where mentioned.